src/app/static/src/app/root.ts

```diff
@@ -127,7 +127,7 @@ declare const currentUser: string;
 
 export const emptyState = (): RootState => {
     return {
-        language: Language.en,
+        language: localStorageManager.getState()?.language || Language.en,
```

Comment:
Because `emptyState` is used in the tests for mocking, I think it would be nice to separate out the fetch from `localStorageManager` from `emptyState` itself, as we do for the modules. 

That would mean leaving `language: Language.en` as the default here, then probably pulling out existing state (again, see modules like `modelOptions.ts`), then on line 154 something like 
```
state: {...emptyState(), ...existingState && {language: existingState.language}}
```

Reply:
I have avoided making language a module, your suggested implementation means I have to make language a module. I believe this will change the existing language structure, which doesn't have language as module even though I do not understand why but I am wary. I can refactor the whole thing and make language work as module, please confirm there was no reason for leaving it as is.

Reply:
I don't believe it would need to be a module -the changes I am suggesting would be in `root.ts`, as described above.

Reply:
Just to be clear: 
```
export const emptyState = (): RootState => {
    return {
        language: Language.en
        ...etc....
    } 
};

const existingState = localStorageManager.getState();

export const storeOptions: StoreOptions<RootState> = {
  state: {...emptyState(), ...existingState && {language: existingState.language}},
   ...etc...
```

Note the difference in the last line with what the modules do - we want to set a single prop value directly on rootState if is existingState is present, not replace all props in a module.

Reply:
Oh in root.ts